.github/workflows/tests.yaml

```diff
@@ -195,7 +197,7 @@ jobs:
                   ./scripts/run_in_build_env.sh \
                   "./scripts/tests/run_test_suite.py \
                      --chip-tool ./out/darwin-x64-chip-tool-${BUILD_VARIANT}${CHIP_TOOL_VARIANT}/chip-tool \
-                     --target-skip-glob '{TestGroupMessaging,TV_*}' \
+                     --target-skip-glob '{TestGroupMessaging,DL_*,TV_*}' \
```

Comment:
Would be better to build door lock app and run the tests against it here, no?

Reply:
I thought that since TV tests were already excluded, DL tests should also be excluded. But I can change that.

Reply:
I have no idea why the TV tests are excluded either, fwiw...

Reply:
I've enabled these test for Darwin platform, unfortunately there is a fail like that:
```
2022-03-04 11:[48](https://github.com/project-chip/connectedhomeip/runs/5421437150?check_suite_focus=true#step:9:48):29.[50](https://github.com/project-chip/connectedhomeip/runs/5421437150?check_suite_focus=true#step:9:50)7 INFO    Each test will be executed 1 times
2022-03-04 11:48:29.508 INFO    Starting iteration 1
2022-03-04 11:48:39.607 ERROR   !!!!!!!!!!!!!!!!!!!! ERROR !!!!!!!!!!!!!!!!!!!!!!
2022-03-04 11:48:39.608 ERROR   ================ CAPTURED LOG START ==================
2022-03-04 11:48:39.608 ERROR   11:48:29.509 - APP       : EXECUTING ['./out/darwin-x64-door-lock-no-ble-asan/chip-door-lock-app', '--discriminator', '[57](https://github.com/project-chip/connectedhomeip/runs/5421437150?check_suite_focus=true#step:9:57)7', '--interface-id', '-1']
2022-03-04 11:48:39.[60](https://github.com/project-chip/connectedhomeip/runs/5421437150?check_suite_focus=true#step:9:60)9 ERROR   ================ CAPTURED LOG END ====================
2022-03-04 11:48:39.609 ERROR   DL_LockUnlock - FAILED in 10.10 seconds
Traceback (most recent call last):
  File "./scripts/tests/run_test_suite.py", line 213, in cmd_run
    test.Run(runner, apps_register, paths)
  File "/Users/runner/work/connectedhomeip/connectedhomeip/scripts/tests/chiptest/test_definition.py", line 232, in Run
    app.start(str(randrange(1, 4096)))
  File "/Users/runner/work/connectedhomeip/connectedhomeip/scripts/tests/chiptest/test_definition.py", line 43, in start
    self.waitForAnyAdvertisement(process, outpipe)
  File "/Users/runner/work/connectedhomeip/connectedhomeip/scripts/tests/chiptest/test_definition.py", line 76, in waitForAnyAdvertisement
    self.__waitFor("mDNS service published:", process, outpipe)
  File "/Users/runner/work/connectedhomeip/connectedhomeip/scripts/tests/chiptest/test_definition.py", line 125, in __waitFor
    raise Exception('Timeout while waiting for %s' % waitForString)
Exception: Timeout while waiting for mDNS service published:
127.0.0.1 - - [04/Mar/2022 11:48:39] "POST / HTTP/1.1" 200 -
Error: The action has timed out.
```

I'm not able to test it on macOS on my side, so I will simply keep this skip glop for now.

Reply:
@Arkq Alright, can you file a followup issue to enable on Darwin too and I will take a look after this lands?

Reply:
OK, I've submitted a new issue for that: https://github.com/project-chip/connectedhomeip/issues/15905